docs/source/en/quicktour.mdx

```diff
@@ -425,6 +403,88 @@ Create a model from your custom configuration with [`TFAutoModel.from_config`]:
 
 Take a look at the [Create a custom architecture](./create_a_model) guide for more information about building custom configurations.
 
+## PyTorch Trainer
+
+All models are a standard [`torch.nn.Module`](https://pytorch.org/docs/stable/nn.html#torch.nn.Module) or a [`tf.keras.Model`](https://www.tensorflow.org/api_docs/python/tf/keras/Model) so you can use them in any standard training loop. While you can write your own training loop, 🤗 Transformers provides a [`Trainer`] class for PyTorch, which contains the basic training loop and adds additional functionality for features like distributed training, mixed precision, and more.
+
+Depending on your task, you'll typically pass the following parameters to [`Trainer`]:
+
+1. A [`PreTrainedModel`] or a [`torch.nn.Module`](https://pytorch.org/docs/stable/nn.html#torch.nn.Module).
+
+        ```py
+        >>> from transformers import AutoModelForSequenceClassification
+
+        >>> model = AutoModelForSequenceClassification.from_pretrained("distilbert-base-uncased")
+        ```
```

Comment:
Nit: here and below, you only need three spaces of indentation (to be at the same level as the A).